File: Assigmment5/Question5.c
#include <stdio.h>
int CheckMax(int iNo1, int iNo2 , int  iNo3)
{
   if (iNo1 > iNo2)
   {
        if (iNo1 > iNo3)
        {
            return iNo1;
        }
        else
        {
            return iNo3;
        }
   }
   else
   {
         if (iNo2 > iNo3)
        {
            return iNo2;
        }
        else
        {
            return iNo3;
        }
   }

}
int main()
{
    int iValue1 = 0, iValue2 = 0 ,iValue3 = 0 , iRet = 0;
    printf("Enter Number 1 : ");
    scanf("%d",&iValue1);
    printf("Enter Number 2 : ");
    scanf("%d",&iValue2);
     printf("Enter Number 3 : ");
    scanf("%d",&iValue3);
    iRet = CheckMax(iValue1 , iValue2, iValue3);

    printf(" %d is the maximum of %d, %d and %d",iRet,iValue1,iValue2,iValue3 );
 

    return 0 ;
}

File: Assignment 19/Question5.java
import java.util.Scanner;

/*
 
2^2= 2 *2
2^3 = 2*2*2
2^4 = 2*2*2*2

 */
class Logic
{
    
    public int DisplayPower(int iNo1 , int iNo2) 
    {
       
        int iCnt = 0 ;
        if (iNo2 ==1)
        {
            return iNo1;
        }

        int iPower =1 ;
        for(iCnt = 1 ; iCnt<=iNo2 ; iCnt++)
        {
            iPower = iPower * iNo1  ;
        }
        return iPower;
       
    }

    
}          
class Question5
{
    public static void main(String[] args) 
    {
        
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0 ,iValue2 = 0 ,iRet = 0;
    
       
        System.out.println("Enter your Number :");
        iValue1 = sobj.nextInt();

         System.out.println("Enter your Power :");
        iValue2 = sobj.nextInt();

        Logic lobj = new Logic();

        iRet = lobj.DisplayPower(iValue1,iValue2);
        System.out.println("The answer is : "+iRet);
        
       
        
    }    
}
